changelog.d/pa-3027.fixed

```diff
@@ -0,0 +1,12 @@
+When `metavariable-type` cannot be evaluated then it defauls to "false", that is,
+it filters out the range. Therefore e.g. this rule:
+
+```yaml
+    patterns:
+      - pattern: private $T x
```

Comment:
This example is confusing to me because I forgot metavariable-type is about checking the identifier
type, not the type itself.
So maybe `private int $X` and the metavariable-type: metavariable: $Y
would be better?
Even though we should really syntactically reject such rules in the metachecker.

Reply:
Ignoring I wrote the wrong tests, I agree this should be caught by the meta checker. I was also thinking whether to write a test at all for this.